Validation/Performance/python/TimeMemoryG4Info.py

```diff
@@ -11,36 +11,36 @@ def customise(process):
     #Tweak Message logger to dump G4cout and G4cerr messages in G4msg.log
     #print process.MessageLogger.__dict__
     #Configuring the G4msg.log output
-    process.MessageLogger.files = dict(G4msg =  cms.untracked.PSet(
-        noTimeStamps = cms.untracked.bool(True)
+    process.MessageLogger.files = dict(G4msg =  dict(
+        noTimeStamps = True
         #First eliminate unneeded output
-        ,threshold = cms.untracked.string('INFO')
-        ,INFO = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,FwkReport = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,FwkSummary = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,Root_NoDictionary = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,FwkJob = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,TimeReport = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,TimeModule = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,TimeEvent = cms.untracked.PSet(limit = cms.untracked.int32(0))
-        ,MemoryCheck = cms.untracked.PSet(limit = cms.untracked.int32(0))
+        ,threshold = 'INFO'
+        ,INFO = dict(limit = 0)
```

Comment:
This level of nesting `dict()`s doesn't work, e.g.
```py
process.MessageLogger.files = dict(G4msg =  dict(
    INFO = dict(limit = 0)
    )
)
```
gives error
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Mixins.py", line 283, in __setattr__
    self.__dict__[name].setValue(value)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Types.py", line 882, in setValue
    setattr(self,k,v)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Mixins.py", line 276, in __setattr__
    self.__addParameter(name, value)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Mixins.py", line 242, in __addParameter
    value = self.__validator.convert_(value)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Types.py", line 81, in convert_
    v = self.__type(value)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Types.py", line 150, in __call__
    return self._pset.clone(**value)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Types.py", line 902, in clone
    _modifyParametersFromDict(myparams, params, self._Parameterizable__raiseBadSetAttr)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Mixins.py", line 750, in _modifyParametersFromDict
    errorRaiser(key)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_3_X_2021-12-12-2300/python/FWCore/ParameterSet/Mixins.py", line 299, in __raiseBadSetAttr
    raise TypeError(name+" does not already exist, so it can only be set to a CMS python configuration type")
TypeError: INFO does not already exist, so it can only be set to a CMS python configuration type
```

On the other hand, the following would work
```py
process.MessageLogger.files = dict(G4msg =  dict(
    noTimeStamps = True
    )
)
process.MessageLogger.files.G4msg.INFO = dict(limit = 0)
```

@Dr15Jones, do you think we could easily overcome this "disparity"?